src/routes/(protected)/settings/+page.svelte

```
@@ -1,39 +1,229 @@
+                                    : "text-muted-foreground hover:bg-muted/50 hover:text-foreground"
+                            )}
+                            onclick={() => handleTabClick(tab.id)}
+                            aria-current={$page.data.activeTabId === tab.id ? "true" : undefined}>
+                            <span>{tab.label}</span>
+                            {#if tab.restartRequired}
+                                <span
+                                    class="bg-amber-500/20 text-amber-600 dark:text-amber-400 rounded px-1.5 py-0.5 text-[10px] font-medium"
+                                    title="Changes require restart">Restart</span>
+                            {/if}
+                        </button>
+                    {/each}
```

ESLint

[error] Each block should have a key

(svelte/require-each-key)